core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/edit-ema-editor.component.ts

```diff
@@ -304,6 +314,15 @@ export class EditEmaEditorComponent implements OnInit, OnDestroy {
     }
 
     handleDragEvents() {
+        this.store.currentState$.pipe(takeUntil(this.destroy$)).subscribe((state) => {
+            if (state === EDITOR_STATE.DRAGGING) {
+                this.iframe.nativeElement.contentWindow?.postMessage(
+                    NOTIFY_CUSTOMER.EMA_REQUEST_BOUNDS,
+                    this.host
+                );
+            }
+        });
```

Comment:
Why did you move it from the dragStart?

Reply:
For perfomance
Before, the request bound was sent in the dragstart

Now with the scroll, during the dragover we can send the request bound or the request scroll

And if we sent the request bound during the dragover, many calls were made and it could affect the performance of the browser

That's why in the dragover only the state changes and the request bound is sent once